components/tikv_util/Cargo.toml

```diff
@@ -60,7 +60,7 @@ sysinfo = "0.26"
 thiserror = "1.0"
 tikv_alloc = { workspace = true }
 time = { workspace = true }
-tokio = { version = "1.5", features = ["rt-multi-thread"] }
+tokio = { version = "1.5", features = ["rt-multi-thread", "full"] }
```

Comment:
@breezewish 
This change is due to files: `components/tikv_util/src/stream.rs:79-80` will throw error `no method named enable_io found for struct `tokio::runtime::Builder` in the current scope` during building.